### PR TITLE
Add-series-content-icon

### PR DESCRIPTION
**Pull request by davidthomas22**
### Context

> Does this issue have a Trello card?

### Intent

> What changes are introduced by this PR that correspond to the above card?

To add the series homepage icon to ensure it is consistent with the other icon designs on the homepage.

> Would this PR benefit from screenshots?

Screenshot 2022-01-12 at 15 57 45

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

File: assets/sass/components/_category.scss [hunk 1]
```diff
@@ -243,22 +243,6 @@
...
-    content: 'SERIES';
-    color: govuk-colour('white');
-    padding: govuk-spacing(1);
-    background-color: govuk-colour('purple');
-    bottom: 0;
-    right: 0;
-    margin-bottom: govuk-spacing(1);
-  }
-}
-
```
Comment (davidthomas22) [inline assets/sass/components/_category.scss:261]: Removed old series image CSS

File: assets/sass/components/_home-content.scss [hunk 2]
```diff
@@ -261,34 +246,3 @@
...
-
-  &::after {
-    background-repeat: no-repeat;
-    content: '';
-    width: govuk-spacing(4);
-    height: govuk-spacing(4);
-    background-image: url('/public/images/icons/chevron-right-white.svg');
-    background-position: 8px 50%;
-  }
-}
```
Comment (davidthomas22) [inline assets/sass/components/_home-content.scss:294]: Removed older content CSS and white chevron icon (/public/images/icons/chevron-right-white.svg) that existed in previous design of the Hub.

**Review by jonhindle — APPROVED** [review]
lgtm

File: assets/sass/components/_home-content.scss [hunk 3]
```diff
@@ -110,11 +106,6 @@
   }
 }
 
-.home-content__four-items > a:hover .content-link--series,
-.home-content__four-items > a:focus .content-link--series {
-  color: govuk-colour('white');
-}
-
```
Comment (davidthomas22) [inline assets/sass/components/_home-content.scss:117]: No longer required as the new icon applies the same SASS properties as all the other icons

**Pull request merged**